### PR TITLE
Update .env.local.example

### PR DESCRIPTION
**Pull request by acidjunk**
#127 Update docs to avoid misleading info about websocket

### CODE REVIEW

File: .env.local.example [hunk 1]
```diff
@@ -1,6 +1,6 @@
 BACKEND_URL="http://localhost:8080"
 # You can use the builtin webppack proxy during development which will connect you to BACKEND_URL
-REACT_APP_BACKEND_URL="http://127.0.0.1:3000"
+REACT_APP_BACKEND_URL="http://127.0.0.1:8000"
```
Comment (pboers1988) [inline .env.local.example:3]: Port 8080 if I'm not mistaken.

**Review by pboers1988 — CHANGES_REQUESTED** [review]

File: .env.local.example (hunk 1 above)
Comment (acidjunk) [reply .env.local.example]: Yes, mea culpa.

**Comment by acidjunk** [comment]
We might change the names a bit to make it more clear. Previously the proxy would also handle request to backend for WS and normal API request. Now we direct them both straight to the backend as the old proxy solution had problems with CRA5

**Review by pboers1988 — APPROVED** [review]
LGTM

**Pull request merged**